yarn.lock

```diff
@@ -3115,14 +3116,6 @@ babel-preset-jest@^29.0.2:
     babel-plugin-jest-hoist "^29.0.2"
     babel-preset-current-node-syntax "^1.0.0"
 
-babel-runtime@~6.25.0:
```

Comment:
We removed a number of dependencies that I don't believe mjml-react needs in production. I'd welcome a PR to @Faire/mjml-react if this isn't the case